packages/engine/tests/experiment/mod.rs

```diff
@@ -202,6 +202,8 @@ pub async fn run_test_suite(
                     output_location: OutputLocation::File("output.log".into()),
                     start_timeout,
                     wait_timeout,
+                    js_runner_initial_heap_constraint: 0,
+                    js_runner_max_heap_size: 4_000,
```

Comment:
Can we add a constant in the JS runner and use this here as well?

Reply:
Changed to `None` in https://github.com/hashintel/hash/pull/446/commits/4e038e0f7466ff82efb87787951fef2af8b8945b

Reply:
Discussed this offline, that using `None` here and fallback to the defaults is more intuitive.